README.rst

```diff
@@ -0,0 +1,49 @@
+
+CellML Specification
+====================
+
+This repository will be used by the CellML editors to develop the CellML specification (starting with version 1.2)
+The content has been reworked to include an informal guide embedded within the normative specification text (in version 2.0).
+
+If things work correctly, changes in this repository will be reflected over at: https://cellml-specification.readthedocs.org/.
+
+Building the documentation locally
+----------------------------------
+
+The documentation can be built on your local machine following these instructions.
+The following instructions are suitable for Unix-derived derived operating systems and may need modification for other operating systems.
+
+First clone a local copy of the CellML specification repository::
+
+  git clone https://github.com/cellml/cellml-specification.git
+
+Next setup a virtual environment::
+
+  virtualenv venv_specification
+
+Note: Make sure you are creating a Python 3 virtual environment, if required use the -p flag to set the Python interpreter (for example `virtualenv -p /absolute/path/to/python venv_specification`).
+
+Then activate the virtual environment and instal the required packages::
+
+  source venv_specification/bin/activate
+  pip install sphinx sphinx_rtd_theme
+  pip install -r cellml-specification/requirements.txt
+
```

Comment:
Maybe add something to say that they will also need to use the python3 version of pip?  Mine is called pip3, not sure if that's standard though?

Reply:
No need for that if you use `virtualenv`. So, if anything, do `pip3 install virtualenv` and then do what is mentioned in the `README.rst` file.

Reply:
I imagine that anyone who has used this stuff since anytime before the last year will already have an installation of virtualenv, as well as both installations of pip?  Even from within a virtual env which specifies python3 I need to also use pip3 to have it work as expected.

Reply:
I have removed my system `pip`. So, all I have is `pip3` on my system. All I did was to install `virtualenv` and then follow the instructions (incl. `pip`).

Reply:
Something is not right with your setup if this is truly the case.  Once activated your virtual environment should have `pip` available.

Reply:
Yep, my bad, forgot to activate second time around.  As long as you put the comment about python3 *before* the instruction to use it, I'm happy now :)